tests_metricflow/snapshots/test_common_dataflow_branches.py/str/test_shared_metric_query__result.txt

```diff
@@ -0,0 +1,160 @@
+test_name: test_shared_metric_query
+test_filename: test_common_dataflow_branches.py
+docstring:
+  For a known case, test that a metric computation node is identified as a common branch.
+
+      A query for `bookings` and `bookings_per_booker` should have the computation for `bookings` as a common branch in
```

Comment:
This is an interesting case! The `ComputeMetricsNode` for `bookings` is common, but _within_ that node there is another common node that could also be used as a CTE - the `ReadSqlSourceNode` for the `bookings_source` table, which is also used in the `bookers` branch. Curious if you plan to add that optimization later?

Reply:
That's correct, and the `ReadSqlSourceNode` is included in the result snapshot under `common_branch_1` below. I should probably add newlines because the different items are easy to miss.

Reply:
Ahh nice! Yep I missed that